modules/virt-creating-a-primary-cluster-udn.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-cluster-udn_{context}"]                                  
+= Creating a primary cluster-scoped user-defined network
+
+You can connect multiple namespaces to the same primary user-defined network (UDN) to achieve native tenant isolation.
+
+.Prerequisites
+. You have access to the cluster as a user with `cluster-admin` privileges.
+. You have installed the OpenShift CLI (`oc`).
+. You have created a namespace and applied the `k8s.ovn.org/primary-user-defined-network` label.
+
+.Procedure
+. Create a `ClusterUserDefinedNetwork` object to specify the custom network configuration:
++
+.Example `ClusterUserDefinedNetwork` manifest
+[source,yaml]
+----
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: cudn-l2-net # <1>
+spec:
+  namespaceSelector: # <2>
+    matchLabels: # <3>
+    - "red-namespace"
+    - "blue-namespace"
```

Comment:
I think this example is not good ... Below we indicate:
> With the `matchLabels` selector, namespace values are evaluated with an _AND_ relationship.

Which means this will match any namespaces with *both* `red-namespace` **and** `blue-namespace` labels in them. 

I don't know, while semantically correct, I think a namespace can't be both red and blue ... That's purple ... 

But this is opinionated! Take it with a grain of salt.

Reply:
I can confirm this is true when `matchLabels` are used.
If one would like to select `red-namespace` and `blue-namespace` namespaces they could use `matchExpressions`:
```
matchExpressions:
- key: kubernetes.io/metadata.name
  operator: In
  values: [ "red-namespace", "blue-namespace" ]
```

Reply:
In SND docs PR there are two examples for CUDN, one demonstrate using `matchLabels` and another one using `matchExpressions`, please see https://github.com/openshift/openshift-docs/pull/85165/files#diff-99caca4d20a24b77fac646362338d4957b4c028f63f8723326924a13b6bbebeaR86-R89

Reply:
Thanks @ormergi .

@sjhala-ccs I think the correct example to show the user (if we're indeed going for "color" based examples) would be the `matchExpressions` one.